docker-build

```diff
@@ -38,22 +47,52 @@ docker_pull() {
 
 pull() {
   local repo="$1"
-  docker_pull "$repo:$BRANCH" || docker_pull "$repo:master" || docker_pull "$repo:latest" || echo "$repo"
+  docker_pull "$repo:$BRANCH" \
+    || docker_pull "$repo:master" \
+    || docker_pull "$repo:latest" \
+    || echo "$repo"
```

Comment:
If we get to this point do we want it to error out?

Reply:
I don't think so, because it'll always hit this on a first build.

Reply:
I think that might make sense if it were being used on it's own, e.g. `docker-build pull`. It's important that it doesn't error for the usage in `--cache-from` [on line 93](https://github.com/remind101/docker-build/blob/f086fe41efede6bae6099c3da467315a2c6a6fdb/docker-build#L93) though.